project/MimaExcludes.scala

```diff
@@ -1202,6 +1206,7 @@ object MimaExcludes {
   }
 
   def excludes(version: String) = version match {
+    case v if v.startsWith("2.5") => v25excludes
```

Comment:
Do we need to update [MimaBuild.scala](https://github.com/apache/spark/blob/master/project/MimaBuild.scala#L91) in this PR together?
```scala
val previousSparkVersion = "2.2.0"
```

Reply:
Sure

Reply:
This needs to be addressed in a separate PR.

Reply:
Oh, I got it, @gatorsmile . Thanks.